parquet-column/src/main/java/org/apache/parquet/column/values/bloomfilter/BlockSplitBloomFilter.java

```diff
@@ -43,11 +42,12 @@
   // Bits in a tiny Bloom filter block.
   private static final int BITS_PER_BLOCK = 256;
 
-  // Default minimum Bloom filter size, set to the size of a tiny Bloom filter block
-  public static final int DEFAULT_MINIMUM_BYTES = 32;
+  // the lower bound of Bloom filter size, set to the size of a tiny Bloom filter block
```

Comment:
Yes, It is intentional. For the maximum value 20MB, it is the size for a row group of 128 MB with only one long type column. Since we have a default maximum value, so I agree to increase the upper bound to a larger value, how about 256MB?

For the minimum value, I think the bloom filter maybe use for other cases in the future other than just row group filter, so set it to a minimum value that could work. I 'm OK to set to a larger value, how about 1k?

Reply:
I am not an expert on bloom filters and I think defining good defaults is not easy. However, this is about to define a range not an exact value so it does not seem to be that though.
My concern was only the fact that the hard limits are the same as the defaults which is strange. If it is intentional that we allow the user to only narrow the range then it is fine however, the reason should be documented. Otherwise, the hard limits should be because of technical limitations (e.g. memory/time limits, meaningless values like -1) and default limits should be practical values for the most cases.

Reply:
The default maximum is 1MB,  and the upper bound is 20MB.  You are right about the upper bound, I think I should set upper bound to row group size since a row group bloom filter should not larger than the row group itself. Does that make sense to you?